vnext/Microsoft.ReactNative/Views/TextInputViewManager.cpp

```diff
@@ -544,6 +581,14 @@ void TextInputShadowNode::updateProperties(winrt::Microsoft::ReactNative::JSValu
       } else if (m_isTextBox != true && react::uwp::IsValidColorValue(propertyValue)) {
         setPasswordBoxPlaceholderForeground(passwordBox, propertyValue);
       }
+    } else if (propertyName == "clearTextOnSubmit") {
+      if (propertyValue.Type() == winrt::Microsoft::ReactNative::JSValueType::Boolean)
```

Comment:
It's a very unlikely corner case, but we may need to check for `propertyValue.IsNull()` and set the `m_clearTextOnSubmit` field to false. This would only occur if a subsequent render of TextInput removed the prop (rather than setting it to false)

Reply:
There's actually a few other props that do not handle this case as well (e.g., `selectTextOnFocus`, `clearTextOnFocus`)

Reply:
#wontfix